src/components/profile/my-recipes-item.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import {useLocation} from "react-router";
+import {useSelector, useDispatch} from "react-redux";
+import {Link} from "react-router-dom";
+import {deleteMyRecipe} from "../../reducers/my-recipes-reducer";
+import "./index.css";
+
+const MyRecipesItem = ({recipe}) => {
+    let loggedIn = useSelector(state => state.profile);
+    const {pathname} = useLocation();
+    const paths = pathname.split('/');
+    let isMyRecipes = paths.includes(loggedIn._id);
+
+    const dispatch = useDispatch();
+    const deleteRecipeHandler = (id) => {
+        dispatch(deleteMyRecipe(id));
+    }
+
+    return (
+        <>
+        {
+            (recipe.privacy === "PUBLIC" || isMyRecipes) &&
+            <div className="m-3 p-2 wd-my-recipe-content">
+                <div className="row d-flex align-items-center ps-2 pe-2">
+                    <span className="col-2">{recipe.privacy}</span>
+                    <div className="col-5 d-flex justify-content-end">
+                        <div className="btn d-block d-xl-none"
+                             onClick={() => deleteRecipeHandler(recipe._id)}>
+                            <i className="bi bi-trash p-2 text-dark wd-text-xs"></i>
+                            <span className="wd-text-xs">Delete Recipe</span>
+                        </div>
+                        <div className="btn d-none d-xl-block"
+                             onClick={() => deleteRecipeHandler(recipe._id)}>
+                            <i className="bi bi-trash text-dark p-2"></i>
+                            <span className="wd-text-md">Delete Recipe</span>
+                        </div>
+                    </div>
+                    <div className="col-5 d-flex justify-content-end">
+                        <div className="btn d-block d-xl-none">
+                            <Link to="./details" className="wd-no-decor">
```

Comment:
Can you change the path for the link to include the recipe ID? So we know which recipe details we are opening